### ISSUE TITLE
Discussion: Future Direction

### ISSUE THREAD

**Issue by Glavin001** [Glavin001/MyoMedia#15]
In the spirit of IoT and to combat the features that Xbox One and others already provide, here is a new direction I propose. Improvements on the following features:
1) Output/Displays:
Xbox One goes on your TV. Chromecast is our TV outlet. However, what if we support more than just TV (chromecast)? How about iOS (iPhone + iPad) and desktop player (Mac, Windows, Linux, etc)
2) Interfaces / Interaction:
Gestures and Voice is great. But how can we be different. Support multiple microphones! Walk around your house and you have many computers. My Raspberry pi has a microphone. My Mac has a microphone. My iphone has a microphone. Why not trigger them ALL to listen? and the pick the most confident result from Wit.ai to be the action. Thus you could simply walk around your house and talk and a different mic would be used when applicable.
3) Intelligent querying of all of your media:
We index it more efficiently and have much more natural querying with Wit.ai. Also querying is more complex. And let's add in history, analytics, recommendation systems. Read https://github.com/Glavin001/MyoMedia/issues/11 
What do you think so far?

**Comment by Glavin001** [comment]
From @siddharthjn for question `So what features do you think are best? MVP, refocus`:

> Music, movie from laptop to tv with smart feedback from tv
> That is core
> 2nd priority should be to figure out a way to do it from mobile
> 3rd priority should be , do you want me to search this on youtube, or do you want me to buy it for you
> Well 3rd is kind of extension of smart feedback
> If 2nd priority is not doable in given time, we can focus on making the feedback smarter once 1st is done
> It would be nice to have the reaearch done for 2nd one, so that we can present it with confidence our future vision
> Which is seamlessly interacting with your devices
